views/forms/SignInForm/index.jsx

```diff
@@ -1,27 +1,35 @@
 import React from 'react'
-import TextInput from '../TextInput'
 import { Button } from '@mui/material'
-import Form from '../Form'
+import { useForm } from 'react-hook-form'
+
+import TextInput from '../TextInput'
 import './SignInForm.css'
 
-const SignInForm = ({ onSubmit }) => {
+const SignInForm = ({ initialValues, onSubmit }) => {
+  const { control, handleSubmit } = useForm({
+    defaultValues: initialValues,
+  })
+
```

Comment:
This feels like it's out of the pattern that we've enforced. Where the parent passes down these props. Why is this changing..?

Reply:
the parent is passing down the initial values and the onSubmit function, it doesn't need to know about the react hook form specifics.